jemalloc-ctl/src/stats_print.rs

```diff
@@ -0,0 +1,165 @@
+//! Bulk statistics output.
+
+extern crate std;
+
+use libc::{c_char, c_void};
+use std::any::Any;
+use std::ffi::CStr;
+use std::io::{self, Write};
+use std::panic::{self, AssertUnwindSafe};
```

Comment:
The `AssertUnwindSafe`  and other `panic` features are not available in `core`, so this whole modulo requires `libstd`.